lib/os/cbprintf_complete.c

```diff
@@ -1372,7 +1372,7 @@ static int outs(cbprintf_cb out,
 	return count;
 }
 
-int cbvprintf(cbprintf_cb out, void *ctx, const char *fp, va_list ap)
+int cbvprintf(cbprintf_cb out, void *ctx, const char *format, va_list ap)
```

Comment:
This routine has two definitions. One here, and another in lib/os/cbprintf_nano.c. The one in cbprintf_nano.c should be updated as well.